scripts/security/scan_for_secrets.py

```
@@ -0,0 +1,286 @@
+                        "line": content[: match.start()].count("\n") + 1,
+                        "pattern": pattern_name,
+                        "match": (
+                            match.group(0)[:50] + "..."
+                            if len(match.group(0)) > 50
+                            else match.group(0)
+                        ),
+                    }
+                )
+    except Exception:
+        # Skip binary files and errors
+        pass
```

Ruff (0.14.4)

`try`-`except`-`pass` detected, consider logging the exception

(S110)